LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 im-open
+Copyright (c) 2021 im-open
```

Comment:
```suggestion
Copyright (c) 2024 WTW
```
Actually, the copyright should be WTW for this open-source codebase, no?

Reply:
I think you're right.